src/pages/Homepage.tsx

```diff
@@ -1,3 +1,14 @@
-import { Example } from '../components/Example/Example';
-
-export const Homepage = () => <Example>Hello world!</Example>;
+import { Welcome } from './sections/Welcome/Welcome';
+import { HowDoesItWork } from './sections/HowDoesItWork/HowDoesItWork';
+import { CurrentlyReading } from './sections/CurrentlyReading/CurrentlyReading';
+import { UpcomingEvents } from './sections/UpcomingEvents/UpcomingEvents';
+import { Testimonials } from './sections/Testimonials/Testimonials';
+export const Homepage = () => (
+  <div>
+    <Welcome />
+    <HowDoesItWork />
+    <CurrentlyReading />
+    <UpcomingEvents />
+    <Testimonials />
+  </div>
```

Comment:
Even we have a lot of space on the size. The content for the text is quite narrow.

![CleanShot 2024-11-05 at 21 46 00](https://github.com/user-attachments/assets/17c77784-b3d3-47ac-a968-f3214066e31a)

I will leave the design up to you. I will only raise stuff like this. The choice is yours. We might want to avoid breaking the `buk klab` heading